identity/client/playwright.config.ts

```diff
@@ -0,0 +1,29 @@
+import { defineConfig, devices } from "@playwright/test";
+
+const IS_CI = Boolean(process.env.CI);
+
+export default defineConfig({
+  testDir: "./e2e",
+  expect: {
+    timeout: 10000,
+  },
+  fullyParallel: true,
+  forbidOnly: IS_CI,
+  retries: IS_CI ? 2 : 0,
+  workers: IS_CI ? 1 : undefined,
+  reporter: "html",
```

Comment:
This has some extra options that are useful. The GH one returns that the results straight into the PR maybe you can remove the `junit` one, it was mostly used for uploads to Testrail which I guess you don't have here

```suggestion
  reporter: IS_CI
    ? [
        ['github'],
        ['html'],
        [
          'junit',
          {
            outputFile: 'results.xml',
          },
        ],
      ]
    : 'html',
```